compiler/AST/primitive.cpp

```diff
@@ -702,11 +702,12 @@ initPrimitive() {
   prim_def(PRIM_REQUIRE, "require", returnInfoVoid, false, false);
 }
 
-Map<const char*, VarSymbol*> memDescsMap;
+static Map<const char*, VarSymbol*> memDescsMap;
+static Map<int, VarSymbol*> memDescsNodeMap;
```

Comment:
Isn't the argument key of this map really a TypeSymbol* ?
I don't think we ever remove TypeSymbols. Am I wrong?